packages/astro/src/integrations/hooks.ts

```diff
@@ -367,6 +372,14 @@ export async function runHookServerSetup({
 	server: ViteDevServer;
 	logger: Logger;
 }) {
+	let refreshContent: undefined | ((options: RefreshContentOptions) => Promise<void>);
+	if (config.experimental?.contentLayer) {
+		refreshContent = async (options: RefreshContentOptions) => {
+			const contentLayer = await globalContentLayer.get();
+			await contentLayer?.sync(options);
+		};
```

Comment:
Is there a way to pass the function only if there are content collections? Or make it a no-op, since we don't have any collections in the data store. 

If we make it a no-op, we can always pass a function regardless, so the type of the downstream integrations won't change.

Reply:
Done